lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -118,6 +119,8 @@ private void init(@Nullable AttributeSet attrs) {
       addColorFilter(new SimpleColorFilter(ta.getColor(
           R.styleable.LottieAnimationView_lottie_colorFilter, Color.TRANSPARENT)));
     }
+
+    scale = ta.getFloat(R.styleable.LottieAnimationView_lottie_scale, Float.MAX_VALUE);
```

Comment:
Move the scale code around line 373 to a helper method and make that the default value here. Then you can remove it from below and just call `setScale(scale)`